### PR TITLE
Enable to specify registry backend

### PR DESCRIPTION
**Pull request by spesnova**
## WHAT

Enable to specify registry backend via environment variables.
- `REGISTRY_BACKEND`: `localfs` or `etcd`
- `REGISTRY_ENDPOINT`:
  - `localfs`: directory path that build data files are stored in (c.f. `/tmp/risu`)
  - `etcd`: etcd's endpoint (c.f. `http://172.17.8.101:4001`)
## WHY

Enable risu user to switch the backend

### CODE REVIEW

**Comment by spesnova** [comment]
@dtan4 review this

File: risu.go [hunk 1]
```diff
@@ -42,7 +44,7 @@ func create(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 		UpdatedAt:      currentTime,
 	}
 
-	reg := registry.NewRegistry("localfs", "")
+	reg.Set(build)
 	err = reg.Set(build)
```
Comment (dtan4) [inline risu.go:47]: duplicated?

**Comment by spesnova** [comment]
updated

**Comment by dtan4** [comment]
```
 _     ____ _____ __  __
| |   / ___|_   _|  \/  |
| |  | |  _  | | | |\/| |
| |__| |_| | | | | |  | |
|_____\____| |_| |_|  |_|
```

**Pull request merged**